uro98/assignment3/FindAllWordsTest.java

```diff
@@ -0,0 +1,66 @@
+import static org.junit.Assert.*;
```

Comment:
Please avoid importing via asterisk.

Reply:
Do you change your imports after you have finished coding or you don't use * at all? Importing * in Eclipse would show recommendations while coding which I think is useful, but I can imagine it'd be hard to remember to change imports afterwards.